HPXMLtoOpenStudio/resources/vehicle.rb

```diff
@@ -0,0 +1,154 @@
+# frozen_string_literal: true
+
+# Collection of methods for adding vehicle-related OpenStudio objects, built on the Battery class
+class Vehicle
+  # Adds any HPXML Vehicles to the OpenStudio model.
+  # Currently only models electric vehicles.
+  #
+  # @param runner [OpenStudio::Measure::OSRunner] Object typically used to display warnings
+  # @param model [OpenStudio::Model::Model] OpenStudio Model object
+  # @param spaces [Hash] Map of HPXML locations => OpenStudio Space objects
+  # @param hpxml_bldg [HPXML::Building] HPXML Building object representing an individual dwelling unit
+  # @param schedules_file [SchedulesFile] SchedulesFile wrapper class instance of detailed schedule files
+  # @return [nil]
+  def self.apply(runner, model, spaces, hpxml_bldg, schedules_file)
+    hpxml_bldg.vehicles.each do |vehicle|
+      next unless vehicle.id.include?('ElectricVehicle')
```

Comment:
Replace with `vehicle_type`. We should also add a warning to the Schematron if the HPXML has a Vehicle that is not of type BEV since we won't model it. (And add a test in test_validation.rb to demonstrate the warning is hit.)